ProjectTemplates/templates/ComponentsWebAssembly-CSharp/Server/Controllers/CallWebApiController.cs

```diff
@@ -11,11 +11,9 @@ namespace AspNetCoreMicrosoftIdentityWebProjectTemplates.templates.ComponentsWeb
     [Authorize]
     [ApiController]
     [Route("[controller]")]
+    [RequiredScope("access_as_user", IsReusable = true)] // The web API will only accept tokens 1) for users, and 2) having the "access_as_user" scope for this API
```

Comment:
As proposed in the other PR, let's embed IsReusable=true in the RequiredScope attribute itself (as this is a perf optimization)

We need to decide if we prefer:
```CSharp
[RequiredScope("access_as_user"] // The web API will only accept tokens 1) for users, and 2) having the "access_as_user" scope for this API
```

or 
```CSharp
[RequiredScope(CallWebApiController.scopeRequiredByApi]
public class CallWebApiController : Controller
    {
        // The web API will only accept tokens 1) for users, and 2) having the "access_as_user" scope for this API
        public const string[] scopeRequiredByApi = new string[] { "access_as_user" };
```

Reply:
fixed in other pr, so will update

---
In reply to: [573604272](https://github.com/AzureAD/microsoft-identity-web/pull/946#discussion_r573604272) [](ancestors = 573604272)